src/applications/disability-benefits/2346/containers/App.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import RoutedSavableApp from '../../../../platform/forms/save-in-progress/RoutedSavableApp';
```

Comment:
Tip: when importing stuff from `platform` or any other directory that lives directly under `src` you can just do `platform/ ... ` without all the leading `../` nonsense :) This is due to this setup in our Babel config https://github.com/department-of-veterans-affairs/vets-website/blob/3fac444474d88b0760fda18339a8130bf017ae3c/.babelrc#L37-L38